### DISCUSSION TITLE
Reduce the size of the Crop Area

### DISCUSSION THREAD

**Discussion by ByteForge2020** [ValentinH/react-easy-crop#528] [Q&A]
When setting { objectFit: "cover" }, what can be done to ensure that the size of the Crop Area is always slightly smaller than the Container? Thank you!

Expected result: 
![2024-04-05_15-07-19](https://github.com/ValentinH/react-easy-crop/assets/57084375/72e8ba51-fe78-404a-9774-0119ea9ca78e)

Current result: 

![2024-04-05_14-53-02](https://github.com/ValentinH/react-easy-crop/assets/57084375/8e4a98d4-a0f3-4521-8921-9c0fdce18b25)

**Comment by ValentinH** [comment]
You might achieve this by doing what's suggested in https://github.com/ValentinH/react-easy-crop/discussions/493#discussioncomment-7352324
